Src/MyCameraPlugin.cs

```diff
@@ -34,6 +34,7 @@ public class MyCameraPlugin : IPluginCameraBehaviour
     // Invoke ApplySettings event when you need to save your settings.
     // Do not invoke event every frame if possible.
     public event EventHandler ApplySettings;
+    private BeatSaberStatus BS;
```

Comment:
Not the best variable naming convention in the world XD

Reply:
Totally fair! x3 That was a holdover from Mori's cam, I agree that it should be updated. BSS it is! (kidding)